docs/plugins/create-a-plugin.md

```diff
@@ -8,7 +8,7 @@ A Backstage Plugin adds functionality to Backstage.
 
 ## Create a Plugin
 
-To create a new plugin, make sure you've run `yarn install` and installed
+To create a new frontend plugin, make sure you've run `yarn install` and installed
```

Comment:
The `yarn install` is not specific to the frontend plugin. As it is running on the root of repository it is required e.g. to install all dependencies of `backstage-cli` or other packages for the first setup up or after pulling the latest changes.

Reply:
Yeah, this is a little tricky. I was trying to avoid rewriting this whole page. It's sort of like a little tutorial for creating a plugin, specifically a frontend plugin. Do you think also updating the heading to include frontend would help? Or maybe I just need to come up with something better here?

Reply:
I think this is alright 👍. It's indeed all for frontend plugins anyway